packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -680,10 +679,6 @@ export async function pollTransformationJob(jobId: string, validStates: string[]
                 throw new JobStoppedError(response.$response.requestId)
             }
             await sleep(CodeWhispererConstants.transformationJobPollingIntervalSeconds * 1000)
-            timer += CodeWhispererConstants.transformationJobPollingIntervalSeconds
-            if (timer > CodeWhispererConstants.transformationJobTimeoutSeconds) {
-                throw new Error('Job timed out')
```

Comment:
Not necessary to do this; better to poll `GetTransformationJob` for latest status and any failure reason